tower-service/Cargo.toml

```diff
@@ -24,4 +24,4 @@ edition = "2018"
 [dependencies]
 
 [dev-dependencies]
-http = "0.1"
+http = "0.2"
```

Comment:
This seems unrelated?

Reply:
Cargo deny was complaining, duplicate bytes versions.